src/components/List/ListItem/ListItem.jsx

```diff
@@ -101,7 +101,10 @@ const ListItem = ({
         onClick={handleExpansionClick}
         onKeyPress={({ key }) => key === 'Enter' && handleExpansionClick()}
       >
-        <Icon icon={expanded ? iconChevronUp : iconChevronDown} />
+        <Icon
+          icon={expanded ? iconChevronUp : iconChevronDown}
+          description={expanded ? 'Expand' : 'Close'}
```

Comment:
these strings need to be passed in as translatable props

Reply:
Updated with i18n props